resources/migrate-to-swift.md

```diff
@@ -47,10 +47,34 @@ If you are using Cocoapods to manage your Adobe Experience Platform Mobile SDK d
   pod 'AEPUserProfile'
 ```
 
-Save the `Podfile` and run `pod repo update` to update your local Cocoapods repository.
+Save the `Podfile` and run `pod repo update` to update your local CocoaPods repository.
 
 Once the previous command is complete, run `pod install` or `pod update` to update the application dependencies.
 
+### SPM
+
+You can now also use SPM to manage your Adobe Experience Platform Mobile SDK dependencies. To add the AEP Packages to your application, from the Xcode menu select:
+
+```
+File > Swift Packages > Add Package Dependency...
+```
+
+Enter the URL for the AEP package repositories: 
+
+- AEPCore package: `https://github.com/adobe/aepsdk-core-ios.git`
+- AEPUserProfile package: `https://github.com/adobe/aepsdk-userprofile-ios.git`
+
+When prompted, input a specific version or a range of versions, and choose all the `AEP*` libraries.
+
+Alternatively, if your project has a `Package.swift` file, you can add AEPCore and AEPUserProfile directly to your dependencies:
+
+```ruby
+dependencies: [
+    .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "3.0.0")),
+    .package(url: "https://github.com/adobe/aepsdk-userprofile-ios.git", .upToNextMajor(from: "3.0.0")),
+]
```

Comment:
The dependencies need to also be added to the target 
```
dependencies: [
    .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "3.0.0")),
    .package(url: "https://github.com/adobe/aepsdk-userprofile-ios.git", .upToNextMajor(from: "3.0.0")),
],
targets: [
       .target(name: "YourTarget",
               dependencies: ["AEPCore", "AEPUserProfile"],
               path: "your/path"),
    ]
]
```

Reply:
Do we need to include all modules here?

Reply:
dependencies: [
        .package(name: "AEPCore", url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "3.0.0")),
        .package(name: "AEPUserProfile", url: "https://github.com/adobe/aepsdk-userprofile-ios.git", .upToNextMajor(from: "3.0.0")),
    ],
    targets: [
        .target(
            name: "YourTarget",
            dependencies: [
                .product(name: "AEPCore", package: "AEPCore"),
                .product(name: "AEPIdentity", package: "AEPCore"),
                .product(name: "AEPSignal", package: "AEPCore"),
                .product(name: "AEPLifecycle", package: "AEPCore"),
                .product(name: "AEPServices", package: "AEPCore"),
                .product(name: "AEPUserProfile", package: "AEPUserProfile")    ,            
            ])
    ]

Reply:
Yeah makes sense to include all the targets for AEPCore. Optional - Since target and package names are the same for User Profile, we can just use "AEPUserProfile"